__tests__/integration.test.js

```diff
@@ -82,6 +82,7 @@ describe('/api/articles/:article_id', () => {
           votes: 100,
           article_img_url:
             'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+          comment_count: '11',
```

Comment:
It's best practice to not alter previous tests when you add more functionality, rather add a new test would be best 🧪